src/routes/auth/login.ts

```diff
@@ -17,7 +17,11 @@ interface HasuraData {
   }
 }
 
-async function loginAccount({ body }: Request, res: Response): Promise<unknown> {
+export const AdminSecretHeader = 'X-HBP-Admin-Secret'
```

Comment:
How about naming the header to `x-hbp-admin-secret` in lower cases?

Reply:
And maybe rename this variable from `AdminSecretHeader` to something more descriptive like:

`NameOfAdminSecretHeader` (just an example)

The variable name `AdminSecretHeader` feels like we are setting both the key and value of the header.